pkg/crc/machine/start.go

```diff
@@ -912,6 +912,13 @@ func logBundleDate(crcBundleMetadata *bundle.CrcBundleInfo) {
 }
 
 func ensureRoutesControllerIsRunning(sshRunner *crcssh.Runner, ocConfig oc.Config, preset crcPreset.Preset, bundleInfo *bundle.CrcBundleInfo) error {
+	// Check if the bundle have `/opt/crc/routes-controller.yaml` file and if it has
+	// then use it to create the resource for router controller.
```

Comment:
"router controller" -> "routes controler". I'd add an article as well "for the routes controller".
Apart from this, looks good to me.

Reply:
This was also remarked in the earlier PR; inconsistency in the use of routes and router.